### ISSUE TITLE
can't open in an existing instance as a tab (whatever kind) with --target tab, tab-bg etc

### ISSUE THREAD

**Issue by evanescente-ondine** [qutebrowser/qutebrowser#6982]
**Version info**:

qutebrowser v2.4.0
Git commit: 
Backend: QtWebEngine 5.15.8, based on Chromium 87.0.4280.144
Qt: 5.15.2

**Does the bug happen if you start with `--temp-basedir`?**:
Yes !

**Description**
If there is an existing instance, it is acknowledged by the command (qutebrowser --target ..) but it opens in a new window regardless of the value given to the --targer option).

![Image:](https://imghost.org/images/2022/01/29/essai.png)

**Comment by The-Compiler** [comment]
Works for me. Can you show your `:config-diff`? What does `which qutebrowser` say? Do you have some kind of wrapper script around the real qutebrowser?

**Comment by evanescente-ondine** [comment]
backend = webengine
colors.webpage.darkmode.contrast = 1.0
colors.webpage.darkmode.enabled = true
https://forum.endeavouros.com: content.notifications.enabled = true
https://www.futura-sciences.com: content.notifications.enabled = true
https://www.reddit.com: content.notifications.enabled = true
https://www.youtube.com: content.notifications.enabled = true
https://mail.google.com?extsrc=mailto&url=%25s: content.register_protocol_handler = true
statusbar.position = top
statusbar.show = in-mode
tabs.last_close = close
tabs.show = multiple


~> which qutebrowser
/usr/bin/qutebrowser

**Comment by The-Compiler** [comment]
Ah, I see now that you just run `--target tab`, but without supplying an URL. That seems kind of expected in that case, as you didn't tell qutebrowser to open anything, and the default behavior if you just launch qutebrowser again is to open a new window.

**Comment by evanescente-ondine** [comment]
But he can't just open a new box where he's asked for. It's minor but maybe
that could be fixed.

**Comment by The-Compiler** [comment]
Agreed - however, you can either provide a URL, or run `qutebrowser ":open -t"` instead.
